index.js

```diff
@@ -11,11 +11,22 @@ var createSourceMapLocatorPreprocessor = function(args, logger, helper) {
     }
 
     function inlineMap(inlineData){
+
+      var charset = 'utf-8';
+      if (/^;charset=([^;]+);/.test(inlineData)) {
+        var matches = inlineData.match(/^;charset=([^;]+);/)
```

Comment:
Sure it's okay, it's just extra allocations and more for the gc (negligible) to do on each invocation instead of only once.
But no duplication was mostly what I was after, so I'm a happy camper! :+1: